packages/components/radio/src/radio.styles.ts

```diff
@@ -0,0 +1,103 @@
+import styled from '@emotion/styled';
+import { RadioProps } from './radio.types';
+
+interface RadioStyledProps {
+  shape?: 'circle' | 'square';
+  backgroundColor?: string;
+}
+
+const handleShape = (shape: RadioProps['shape']) => {
+  switch (shape) {
+    case 'circle':
+      return 'border-radius: 50%;';
+    case 'square':
+      return 'border-radius: 0;';
+    default:
+      return 'border-radius: 50%;';
+  }
+};
+
+export const RadioStyled = styled.div<Omit<RadioStyledProps, 'label'>>`
+  display: flex;
+  align-items: center;
+  background-color: ${props => props.backgroundColor || 'transparent'};
+
+  input[type="radio"] {
```

Comment:
Input must be a separated const.

```suggestion
  export const InputStyled = styled.input`...`
```